src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    list-style-type: none;
+  }
+
+  &__item:not(:last-child) {
+    padding-right: 20px;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $color-light-black;
+    display: inline-block;
+    padding-top: 23px;
+    padding-bottom: 22px;
+    font-weight: $medium-font-weight;
+    font-size: $small-font-size;
+    text-transform: uppercase;
+  }
+
+  .is-active {
+    color: $color-accent-blue;
+    padding-bottom: 0;
+  }
+
+  .is-active::after {
+    background-color: $color-accent-blue;
+    content: "";
+    display: block;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    margin-top: 18.5px;
+    cursor: auto;
+  }
+
+  &__link:hover {
+    color: $color-accent-blue;
+  }
```

Comment:
Move it to link element

&:hover {
    color: $color-accent-blue;
  }

Reply:
is not fixed. Mentor meant to apply this piece of code 
```
&:hover {
  color: $color-accent-blue;
}
```
inside  `&__link ` selector